package.json

```diff
@@ -62,7 +62,7 @@
     "lodash": "^4.17.19",
     "mal-scraper": "^2.5.0",
     "mathjs": "^5.7.0",
-    "moment": "^2.24.0",
+    "moment": "^2.29.1",
```

Comment:
it really needs a newer version?

Reply:
Nope, this is my bad. I think I might have done this on accident trying to fix something. I'll revert it.